modules/camera/backend/modules/camera/models.py

```diff
@@ -0,0 +1,23 @@
+from django.db import models
+from django.conf import settings
+
+class Image(models.Model):
+    user = 1
+    title = models.CharField(max_length=128, null=True, blank=True)
+    comment = models.CharField(max_length=512, null=True, blank=True)
+    lat_long = models.CharField(max_length=100, null=True, blank=True)
```

Comment:
Can we please remove all the model fields that are not being used yet? We can include them later when needed